server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -21,6 +21,11 @@ See the radio-option below as an example.
 
 @use 'uswds-core' as *;
 
+.cf-alert {
+  max-width: 1024px;
+  margin: 0 auto;
```

Comment:
Not sure if we'd actually put the rule here, but from the mocks shouldn't this have some vertical margin? Just checking, if it's out of scope no worries.

Reply:
(blocking) It would be better if this could be expressed in the html using the USWDS utility classes. 

`.width-desktop`

https://designsystem.digital.gov/utilities/height-and-width/

Reply:
@devin-civi Yeah, spacing is defined on each page, and if the page hasn't been styled yet, it looks wrong.

Good example: date-filled-in-north-star.png

We'll fix each page individually (aka out of scope of this PR)

@gwendolyngoetz Oh yeah, fixed